### PR TITLE
Release 2 Apps-5

### PR DESCRIPTION
**Pull request by priyansh-sumo**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [ ] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/saas-cloud/druva-cyber-resilience.md [hunk 1]
```diff
@@ -64,15 +64,13 @@ _source="Milan_Druva_Cyber_Resilience_v0.0.1"
...
 To set up [Cloud to Cloud Integration Druva Cyber Resilience Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-cyber-resilience-source) for the Druva Cyber Resilience App, follow the instructions provided. These instructions will guide you through the process of creating a source using the Druva Cyber Resilience source category, which you will need to use when installing the app. By following these steps, you can ensure that your Druva Cyber Resilience app is properly integrated and configured to collect and analyze your Druva Cyber Resilience data.
 
-## Installing the Druva Cyber Resilience app​
+## Installing the Druva Cyber Resilience app
 
-This section has instructions for installing the Druva app for Sumo Logic.
+{@import ../../reuse/apps/app-install-v2.md}
 
-{@import ../../reuse/apps/app-install.md}
+## Viewing Zscaler Private Access dashboards
```
Comment (jpipkin1) [inline docs/integrations/saas-cloud/druva-cyber-resilience.md:71]:
```suggestion
## Viewing Druva Cyber Resilience dashboards​
```

File: docs/integrations/saas-cloud/tenable.md [hunk 2]
```diff
@@ -52,15 +52,13 @@ _sourceCategory=Tenable
...
 To set up [Cloud to Cloud Integration Tenable Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/tenable-source/) for the Tenable app, follow the instructions provided. These instructions will guide you through the process of creating a source using the Tenable source category, which you will need to use when installing the app. By following these steps, you can ensure that your Tenable app is properly integrated and configured to collect and analyze your Tenable data.
 
-## Installing the Tenable app​
+## Installing the Tenable app
 
-This section has instructions for installing the Symantec Web Security Service App for Sumo Logic.
+{@import ../../reuse/apps/app-install-v2.md}
 
-{@import ../../reuse/apps/app-install.md}
+## Viewing Zscaler Private Access dashboards
```
Comment (jpipkin1) [inline docs/integrations/saas-cloud/tenable.md:59]:
```suggestion
## Viewing Tenable dashboards​
```

File: docs/integrations/web-servers/heroku.md [hunk 3]
```diff
@@ -192,21 +192,13 @@ d.98ee476d-d2d8-46bf-afc2-740f6f7e5b2a
...
 These custom app **_sourceNames** will appear as values in the dashboard filter variable ``log_drain`` for the Heroku app dashboards.
 
-## Installing the Heroku App
+## Installing the Heroku app
 
-This section provides instructions for installing the Heroku App.
+{@import ../../reuse/apps/app-install-v2.md}
 
-Now that you have set up collection for Heroku, you can install the Heroku App to use the preconfigured searches and Dashboards that provide insight into your data.
+## Viewing Zscaler Private Access dashboards
```
Comment (jpipkin1) [inline docs/integrations/web-servers/heroku.md:199]:
```suggestion
## Viewing Heroku Dashboards
```

**Review by jpipkin1 — DISMISSED** [review]
@priyansh-sumo, I found a few headings that you need to fix.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**